README.md

```diff
@@ -301,20 +326,25 @@ See more examples in the demo application: [ViewController.swift](/Demo/Common/V
 ![Image](Assets/TwitterOAuthTokens.png "Image")
 
 ## Contributing
- See [CONTRIBUTING.md](.github/CONTRIBUTING.md)
 
-[Add a new service in demo app](https://github.com/OAuthSwift/OAuthSwift/wiki/Demo-application#add-a-new-service-in-demo-app)
+See [CONTRIBUTING.md](.github/CONTRIBUTING.md)
 
+[Add a new service in demo app](https://github.com/OAuthSwift/OAuthSwift/wiki/Demo-application#add-a-new-service-in-demo-app)
 
 ## Integration
+
 OAuthSwift could be used with others frameworks
 
 You can sign [Alamofire](https://github.com/Alamofire/Alamofire) request with [OAuthSwiftAlamofire](https://github.com/OAuthSwift/OAuthSwiftAlamofire)
 
 To achieve great asynchronous code you can use one of these integration frameworks
-- [OAuthSwiftFutures](https://github.com/OAuthSwift/OAuthSwiftFutures) - [BrightFutures](https://github.com/Thomvis/BrightFutures)
-- [OAuthRxSwift](https://github.com/OAuthSwift/OAuthRxSwift) - [RxSwift](https://github.com/ReactiveX/RxSwift)
-- [OAuthReactiveSwift](https://github.com/OAuthSwift/OAuthReactiveSwift) - [ReactiveSwift](https://github.com/ReactiveCocoa/ReactiveSwift)
+
+- [OAuthSwiftFutures](https://github.com/OAuthSwift/OAuthSwiftFutures)
```

Comment:
not sure, here, OAuthSwiftFutures and BrightFutures must be aligned

Reply:
Ah, I see. These pairs of links were meant to be on the same line.

![Screen Shot 2021-05-19 at 10 58 13 AM](https://user-images.githubusercontent.com/28851/118853627-2e362080-b891-11eb-9453-600fdd6d77eb.png)

I'll restore this.